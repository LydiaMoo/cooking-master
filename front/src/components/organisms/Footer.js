import React, { Component } from 'react';
// import styled from 'styled-components';

const endFooterStyling = {
background: '#635155',
padding: '2%',
display: 'flex',
justifyContent: 'center',
color: 'white', 
}

export default class Footer extends Component {
  render() {
    return (
      <footer style={endFooterStyling} >
      Made by {this.props.name}
      </footer> 
    )
  }
}
