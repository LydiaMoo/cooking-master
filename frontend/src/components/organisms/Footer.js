import React, { Component } from 'react';
import styled from 'styled-components';

const EndFooter = styled.footer`
background: #635155;
padding: 2%;
display: flex;
justify-content: center; 
color: white; 
`;

export default class Footer extends Component {
  render() {
    return (
      <EndFooter>
      Made by {this.props.name}
      </EndFooter> 
    )
  }
}
