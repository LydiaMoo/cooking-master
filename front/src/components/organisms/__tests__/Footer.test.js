import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Footer from '../Footer';

describe('<Footer />', () => {

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Footer name="Percy" />, div);
  });

  it('should match default snapshot', () => {
    const tree = renderer.create(
      <Footer />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});