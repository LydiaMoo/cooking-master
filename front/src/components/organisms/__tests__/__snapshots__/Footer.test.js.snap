// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match default snapshot 1`] = `
<footer
  style={
    Object {
      "background": "#635155",
      "color": "white",
      "display": "flex",
      "justifyContent": "center",
      "padding": "2%",
    }
  }
>
  Made by 
</footer>
`;
